tests/sav/syntax.res

```diff
@@ -0,0 +1 @@
+../examples/syntax.nit:46,2--15: Error: dev package for `c` unknown by `pkg-config`, install it with `apt-get`, `brew` or similar.
```

Comment:
Or with an existing pkg-config package so there is no error! something like `bdw-gc`, `libunwind` or `sqlite3`.